R/pkg/R/mllib.R

```diff
@@ -99,10 +114,8 @@ setMethod("spark.glm", signature(data = "SparkDataFrame", formula = "formula"),
             return(new("GeneralizedLinearRegressionModel", jobj = jobj))
           })
 
-#' Fits a generalized linear model (R-compliant).
-#'
-#' Fits a generalized linear model, similarly to R's glm().
-#'
+#' @title Generalized Linear Models (R-compliant)
```

Comment:
Also, as talked with this discussion https://github.com/apache/spark/pull/13394#discussion-diff-66177097, should we follow first sentence as the convention to define the title in this case?